sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -32,7 +34,19 @@ initscript {
         logger.quiet("Develocity plugins resolution: $pluginRepositoryUrl")
 
         repositories {
-            maven { url pluginRepositoryUrl }
+            maven {
+                url pluginRepositoryUrl
+                if (pluginRepoUsername && pluginRepoPassword) {
+                    logger.quiet("Using credentials for plugin repository")
```

Comment:
With f091a59e673b494634fcadd853e89a57a363351e merged I think this should be logged at `lifecycle`.